v5/v5_stable.ino

```diff
@@ -133,6 +137,7 @@ int allowed_debugging[7] = {0, 0, 1, 0, 1, 0, 1};
 // (debuggerID == 4) // debugging WHATSAPP_debug functionality
 // (debuggerID == 5) // debugging BLE_debug functionality
 // (debuggerID == 6) // debugging SPIFFS_debug functionality
+// (debuggerID == 7) // debugging Orange-pi functionality
```

Comment:
I new task is assigned kindly mark that too.

Reply:
kindly mention it, or write details here, as this part is doing its work